### PR TITLE
feat: lab contract write example

### PR DESCRIPTION
**Pull request by tomiir**
# Breaking Changes

N/A

# Changes

- feat: adds contract write tests by calling a Donut Vending Machine contract deployed on sepolia 
- fix:
- chore:

### CODE REVIEW

**Review by svenvoskamp — APPROVED** [review]
Approved if merge conflicts are fixed

File: .vscode/settings.json [hunk 1]
```diff
@@ -10,9 +10,9 @@
   "files.exclude": {
     "**/.DS_Store": true,
     "**/.git": true,
-    "**/node_modules": true,
-    "**/out": true,
-    "**/dist": true,
+    "**/node_modules": false,
+    "**/out": false,
+    "**/dist": false,
```
Comment (0xAsimetriq) [inline .vscode/settings.json:15]: Do we need to update this, or can this be kept as local vscode override? Did you need to access these folders (they should never be tweaked as they are products of build).

**Review by 0xAsimetriq — APPROVED** [review]

File: .vscode/settings.json (hunk 1 above)
Comment (tomiir) [reply .vscode/settings.json]: added .vscode folder to gitignore since it's not supposed to change 🙏

**Review by arein — APPROVED** [review]
we need to spin up a repo for the donut contract later and CI etc

**Pull request merged**